modules/data-catalog-policy-tag/variables.tf

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "activated_policy_types" {
+  description = "A list of policy types that are activated for this taxonomy. See the [documentation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/data_catalog_taxonomy#activated_policy_types) for possible values."
```

Comment:
move the link to the README, as it will not be rendered correctly in every situation here